src/App.tsx

```diff
@@ -1,24 +1,187 @@
-/* eslint-disable max-len */
```

Comment:
Please remove the unnecessary eslint-disable rule for 'max-len' as it is not required in this file.